components/navbar/ProfileDropdown.vue

```diff
@@ -30,12 +30,16 @@
               {{ $t('credit') }}
             </a>
           </b-dropdown-item>
-          <b-dropdown-item has-link aria-role="menuitem">
-            <nuxt-link :to="{ name: 'identity' }">
-              {{ $t('identity.page') }}
-            </nuxt-link>
-          </b-dropdown-item>
         </template>
+        <b-dropdown-item
+          v-if="chain === 'rmrk' || chain === 'ksm' || chain === 'dot'"
```

Comment:
why remove `bsx` and `snek` ? should be available on all chain (maybe not Moonbean)